cmake/Findsndfile.cmake

```diff
@@ -0,0 +1,32 @@
+include(FindPackageHandleStandardArgs)
+
```

Comment:
Make sure to install this file, by adding it to
https://github.com/libsdl-org/SDL_mixer/blob/2b62bf7f5b60544b1bfc65a2d5568a79d7fe6a92/CMakeLists.txt#L894-L911